dashbuilder-website-docs/src/main/asciidoc/guides/yaml_guide.adoc

```diff
@@ -1518,4 +1694,11 @@ pages:
                     uuid: test
 ----
 .Dark Mode
-image::guides/yaml/darkMode.png[Dark Mode]
\ No newline at end of file
+image::guides/yaml/darkMode.png[Dark Mode]
+
+There are new fields which can be added to Global properties; they are as  folows:-
```

Comment:
Please avoid using *new*, try something like: "The global properties are:". Also there's a lost hyphen at the end of the sentence.